### PR TITLE
Update for SR2019

### PR DESCRIPTION
**Pull request by Scarzy**
Fixes https://github.com/srobo/tasks/issues/66

### CODE REVIEW

**Comment by PeterJCLaw** [comment]
@Scarzy are you looking for review of the content here, or is this PR more about getting the content merged?

File: competition.tex [hunk 1]
```diff
@@ -44,56 +41,67 @@ \section{Risks}
...
+
 \risk{Battery failure - smoke, fire}
-{Teams will hand over batteries and chargers on arrival, from this point
-SQEP Blueshirts to handle battery charging in a designated area. Blueshirts
-and Team Leaders to identify batteries showing signs of damage or swelling
-and deliver to Helpdesk for safe disposal.}
-{SC - Events, Team Leaders}
+{All batteries to be charged by trained volunteers in fire-proof bags.
+Competitors and team leaders have been informed about safe use of the batteries throughout the competition year.
+Blueshirts and Team Leaders to identify batteries showing signs of damage or swelling and deliver to Helpdesk for safe disposal.}
```
Comment (PeterJCLaw) [inline competition.tex:86]: Hrm, this should impact https://github.com/srobo/tasks/issues/122 (helpdesk procedures) -- we should have a means there for the safe storage/disposal of any bad batteries (see also https://github.com/srobo/tasks/issues/121, battery charging procedure)

**Review by PeterJCLaw — COMMENTED** [review]
Content looks good & this builds fine locally.

Do we need to do anything with regards to a general risk assessment of the venue, or is that out of scope? I note that previously Newbury's own general risk assessment was referenced - is there an equivalent it makes sense for us to reference?

While I don't know what the concrete plan for the arena lighting/sound is (https://github.com/srobo/tasks/issues/74, https://github.com/srobo/tasks/issues/75), is there anything there we should account for?

Is it within the scope of this document to consider the carrying of robots specifically into and out of the arena, which involves climbing through the scaff?

**Comment by Scarzy** [comment]
> Do we need to do anything with regards to a general risk assessment of the venue, or is that out of scope? I note that previously Newbury's own general risk assessment was referenced - is there an equivalent it makes sense for us to reference?

Yes, if we can get a copy. 
 
> While I don't know what the concrete plan for the arena lighting/sound is ([srobo/tasks#74](https://github.com/srobo/tasks/issues/74), [srobo/tasks#75](https://github.com/srobo/tasks/issues/75)), is there anything there we should account for?

Hmm... Probably!

> Is it within the scope of this document to consider the carrying of robots specifically into and out of the arena, which involves climbing through the scaff?

Not a bad shout, though it might be merged into another, more general item.

**Comment by Scarzy** [comment]
> > While I don't know what the concrete plan for the arena lighting/sound is ([srobo/tasks#74](https://github.com/srobo/tasks/issues/74), [srobo/tasks#75](https://github.com/srobo/tasks/issues/75)), is there anything there we should account for?
> 
> Hmm... Probably!

Added reference to both.
 
> > Is it within the scope of this document to consider the carrying of robots specifically into and out of the arena, which involves climbing through the scaff?
> 
> Not a bad shout, though it might be merged into another, more general item.

Added as a risk.

**Comment by RealOrangeOne** [comment]
(The CI failures can be fixed by updating from `master`)

**Comment by PeterJCLaw** [comment]
@WillB97 do you happen to know whether SUSU has a general risk assessment for events and if so whether we could have a copy of it?

**Comment by Scarzy** [comment]
> @WillB97 do you happen to know whether SUSU has a general risk assessment for events and if so whether we could have a copy of it?

As that would be an addition to the risk assessment, rather than an amendment, can I suggest this PR is accepted and if/when a SUSU risk assessment arrives I can review and submit a new PR if necessary?

**Review by PeterJCLaw — APPROVED** [review]

**Review by RealOrangeOne — APPROVED** [review]
This looks fine to me, although I have little experience in risk assessments, so don't consider this ready based off just my review.

**Pull request merged**